chart/values.yaml

```diff
@@ -101,6 +102,7 @@ controller:
     image:
       repository: hashicorp/vault-secrets-operator
       tag: 0.3.4
+      pullPolicy: Always
```

Comment:
See the comment related to the kubeRbacProxy container change above.

Reply:
I set it to ""IfNotPresent